alpha/template/semver/types.go

```diff
@@ -4,13 +4,15 @@ import (
 	"io"
 
 	"github.com/blang/semver/v4"
+
 	"github.com/operator-framework/operator-registry/pkg/image"
 )
 
 // data passed into this module externally
 type Template struct {
-	Data     io.Reader
-	Registry image.Registry
+	Data         io.Reader
+	Registry     image.Registry
+	MigrateLevel string
```

Comment:
Nit: I see some fields called `MigrateLevel` and others called `MigrationLevel`. Should we make those consistent?

Only exception to this rule I could see if the `Migrate.Level` field, which makes sense to me to be named differently to avoid stuttering.